Snowdrift.cabal

```diff
@@ -215,9 +215,9 @@ library
                  , time
                  , titlecase
                  , transformers
-                 , unix
```

Comment:
Oops, I was wrong, we *did* add `unix` back in early 2014, but we added it specifically to have env as in `lookupEnv`. So if that is now in base, we do not in fact need unix anymore. So it is indeed fine to remove.